unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Camera/Resources/CameraController.prefab

```diff
@@ -66,7 +66,7 @@ MonoBehaviour:
     ModeOverride: 2
     LensShift: {x: 0, y: 0}
     GateFit: 2
-    m_SensorSize: {x: 1.6825095, y: 1}
+    m_SensorSize: {x: 2.3333333, y: 1}
```

Comment:
We added a new layer to the project, and we need to change the camera to render that layer as well.

Reply:
Why do we need the camera to render that layer? Isn't that layer just for collisions?

Reply:
Because the primitives don't have a  game object, they use the same game object. Also, we are working towards have only 1 collider so we can have the collider and the renderer in the same gameobject

Reply:
So should we change the camera in this PR as well?

Reply:
It should better to have another game object for the collider, I wouldn't mind if unity let us use bitmasks for layer but it is not the case, so for scalability proposes we should put the collider on it own game object